scripts/linux/cross-platform-rust-build.sh

```diff
@@ -70,34 +70,20 @@ case "$PACKAGE_OS" in
     'ubuntu18.04')
         DOCKER_IMAGE='ubuntu:18.04'
         ;;
+
+    'alpine1.19.3')
+        DOCKER_IMAGE='ubuntu:18.04'
+        ;;       
 esac
 
 if [ -z "$DOCKER_IMAGE" ]; then
     echo "Unrecognized target [$PACKAGE_OS.$PACKAGE_ARCH]" >&2
     exit 1
 fi
 
-case "$PACKAGE_ARCH" in
-    'amd64')
-        RUST_TARGET='x86_64-unknown-linux-musl'
-        ;;
-
-    'arm32v7')
-        RUST_TARGET='armv7-unknown-linux-gnueabihf'
-        ;;
-
-    'aarch64')
-        RUST_TARGET='aarch64-unknown-linux-gnu'
-        ;;
-esac
-
-if [ -n "$RUST_TARGET" ]; then
-    RUST_TARGET_COMMAND="rustup target add $RUST_TARGET &&"
-fi
-
-
 case "$PACKAGE_OS.$PACKAGE_ARCH" in
-    ubuntu18.04.amd64)
+    ubuntu18.04.amd64 | alpine1.19.3.amd64)
```

Comment:
Since we aren't building for ubuntu18.04, I think we should change it to alpine in our build scripts.

Reply:
I agree but this close to the release I don't feel confident changing to 100 different places :X